src/proto_tcp.c

```diff
@@ -81,6 +81,9 @@ struct protocol proto_tcpv4 = {
 	.proto_type     = PROTO_TYPE_STREAM,
 	.sock_type      = SOCK_STREAM,
 	.sock_prot      = IPPROTO_TCP,
+#ifdef __linux__
+	.sock_prot_mptcp= IPPROTO_MPTCP,
```

Comment:
I don't think you need this. You need to modify `sock_prot` if the option is set.